packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -1283,6 +1293,46 @@ export class ZWaveController
 		this._nodeType = initData.nodeType;
 		this._supportsTimers = initData.supportsTimers;
 		// ignore the initVersion, no clue what to do with it
+
+		// BUGBUG: Dummy check? SimplicityStudio adds this extra 3 argument to this command. I suspect that's not the secret sauce for LR inclusion, but S2 encryption is.
+		await this.driver.sendMessage<
+			GetSerialApiCapabilitiesResponse
+		>(
+			new GetSerialApiCapabilitiesRequest(this.driver, {mysteryValue: 3}),
+			{
+				supportCheck: false,
+			},
+		);
+
+		// fetch the list of long range nodes until the controller reports no more
+		const lrNodeIds: Array<number> = [];
+		let lrChannel : LongRangeChannel | undefined;
+		const maxPayloadSize = await this.getMaxPayloadSize();
+		let maxPayloadSizeLR : number | undefined;
+		if (this.isLongRange()) {
+			const nodePage = 0;
+			while (true) {
+				const nodesResponse = await this.driver.sendMessage<
+					GetLongRangeNodesResponse
+				>(
+					new GetLongRangeNodesRequest(this.driver, {
+						listStartOffset128: nodePage,
+					}),
+				);
+				lrNodeIds.push(...nodesResponse.nodeIds);
+				if (!nodesResponse.moreNodes) {
+					break;
+				}
+			}
+
+			// TODO: restore/set the channel
```

Comment:
We probably need a driver constructor argument to choose a long range channel to set, yeah?

And maybe TXPower, etc.